src/qt/trafficgraphwidget.cpp

```diff
@@ -153,12 +152,12 @@ void TrafficGraphWidget::updateRates()
     update();
 }
 
-void TrafficGraphWidget::setGraphRangeMins(int mins)
+void TrafficGraphWidget::setGraphRange(std::chrono::minutes mins)
 {
-    nMins = mins;
-    int msecsPerSample = nMins * 60 * 1000 / DESIRED_SAMPLES;
+    m_range = std::chrono::minutes{mins};
```

Comment:
Maybe just
```suggestion
    m_range = mins;
```
?

Reply:
The type-conversion here seems redundant.

Addressed in f7a19ef774ef92ce348215593e3590a750c345e1